### PR TITLE
Added some more useful Annotations to the Example Gallery

### PR DESCRIPTION
**Pull request by kolibril13**
See in Example Gallery

### CODE REVIEW

File: docs/source/examples/annotations.rst [hunk 1]
```diff
@@ -28,6 +28,23 @@ Annotations
...
+            m0 = SmallDot()
+            m1 = AnnotationDot()
+            m2 = LabeledDot("ii")
+            m3 = LabeledDot(MathTex(r"\alpha").set_color(ORANGE))
+            m4 = CurvedArrow(ORIGIN,2*LEFT)
+            m5 = CurvedDoubleArrow(ORIGIN,2*RIGHT)
+
+            self.add(m0,m1,m2,m3,m4,m5)
+            for i,mobj in enumerate(self.mobjects):
+                mobj.shift(DOWN*(i-3))
```
Comment (leotrs) [inline docs/source/examples/annotations.rst:46]:
```suggestion
            m4 = CurvedArrow(ORIGIN, 2*LEFT)
            m5 = CurvedDoubleArrow(ORIGIN, 2*RIGHT)

            self.add(m0, m1, m2, m3, m4, m5)
            for i, mobj in enumerate(self.mobjects):
                mobj.shift(DOWN * (i-3))
```

**Review by leotrs — APPROVED** [review]

**Comment by leotrs** [comment]
Just committed a minor change.

File: docs/source/examples/annotations.rst [hunk 2]
```diff
@@ -28,6 +28,23 @@ Annotations
             tip_text = Text('(2, 2)').next_to(arrow.get_end(), RIGHT)
             self.add(numberplane, dot, arrow, origin_text, tip_text)
 
+.. manim:: UsefulAnnotations
+    :quality: medium
```
Comment (behackl) [inline docs/source/examples/annotations.rst:32]: The quality used for rendering the directive without options is optimized for the documentation already. Is there a reason for using `:quality: medium` here? If not, I'd suggest to remove it:

```suggestion
```

**Review by behackl — CHANGES_REQUESTED** [review]
LGTM, thank you! Just one minor comment, it can be merged as soon as this is resolved.

(The example also fits with respect to the suggested guidelines for separating examples from #591.)

File: docs/source/examples/annotations.rst (hunk 2 above)
Comment (kolibril13) [reply docs/source/examples/annotations.rst]: nope, there is not :)
I just copy pasted it from another example, were we also still have a `    :quality: medium` flag

**Review by leotrs — APPROVED** [review]

**Pull request merged**